components/Slider/src/MDCSlider.h

```diff
@@ -47,22 +49,35 @@ IB_DESIGNABLE
 
  Default color is blue.
  */
-@property(nonatomic, strong, null_resettable) UIColor *color;
+@property(nonatomic, strong, null_resettable) UIColor *color UI_APPEARANCE_SELECTOR;
```

Comment:
`null_resettable` UIAppearance-proxied properties are pretty rare and a discouraged pattern for MDC. Does it make sense to change them to `nullable` either here or in another PR?

Reply:
+1ing this (deleting previous comment)

Reply:
Filed #2815.